Journal of Ethics in Antiquity and Christianity.js

```diff
@@ -0,0 +1,96 @@
+{
+	"translatorID": "9e186c48-48e4-4cbc-be69-289bda8359ad",
+	"label": "Journal of Ethics in Antiquity and Christianity",
+	"creator": "Madeesh Kannan",
+	"target": "https?://jeac.de/ojs/index.php",
```

Comment:
Target passt nicht zu `detectWeb` und der Test-Url. Warum ist dieser Translator überhaupt nötig?

Reply:
Unter diesem Namen ist er aber vollständig neu. Was es offebar bereits gibt, ist `ubtue_JEAC.js`. Hier wird für den Orcid-Fall aber auch der falsche Sub-Translator aufgerufen (Open Journal System Standard).

Reply:
`Journal of Ethics in Antiquity and Christianity.js` sollte auf jeden Fall wieder aus dem PR raus.

Reply:
Nein. Es kann gelöscht werden. JEAC kann mit "ubtue_Open Journal System Standard" heruntergeladen werden.

Reply:
Lieber @jriedl , offensichtlich arbeite ich mit einem komplett falschen Repositorium, ich kann mir nicht erklären, wie das sonst zustande kommt. Ich wüsste nicht, wie der Translator sonst da hineinkommen sollte. Wir sollten am Montag mit @socheres klären, inwiefern meine Mitarbeit in diesem Projekt überhaupt effizient ist.